### DISCUSSION TITLE
Discord RPC, New hacks and fixes

### DISCUSSION THREAD

**Discussion by prevter** [Prevter/GDOpenHack#21] [Announcements]
Changes:
- Added "Discord Rich Presence"
- Fixed DLL loader to load before the game starts
- "Unlock All Icons" now saves your icons if you do not disable it
- "Music Unlocker" now also unlocks Practice Mode music
- Added descriptions (tooltips) to almost all hacks
- Various fixes to old hacks

New hacks:
**Bypass**: "Allow Low Volume", "Fast Chest Open"
**Cosmetic**: "No Orb Ring", "No Glow", "No Wave Trail", "Trail Cutting", "Forever Trail"
**Creator**: "Free Scroll"
**Level**: "No Rotation", "Force Visibility", "No Shaders", "Pause During Complete"
**Universal**: "Free Window Resize"

<hr /><em>This discussion was created from the release <a href='https://github.com/Prevter/GDOpenHack/releases/tag/1.3.0'>Discord RPC, New hacks and fixes</a>.</em>

**Comment by enaito4545** [comment]
plis startpose switcher
